docs/user-guide/src/capm3/node_reuse.md

```diff
@@ -0,0 +1,45 @@
+# Node Reuse
+
+This feature brings a possibility of re-using the same BaremetalHosts (referred to as a host later)
+during deprovisioning/provisioning mainly as part of the rolling upgrade process in the cluster.
```

Comment:
Is this part trying to answer _why we need it?_  just thinking that we might need to emphasize that need for using the same external storage

Reply:
What is the external storage in this context, do you mean m3 object, i.e m3mt?

Reply:
I mean as a necessity that node reuse cover. as we have an external storage attached to a node we prefer reusing the same node and keep the external storage I guess that is one of the main reason to implement nodereuse, isn't it?

Reply:
Yeah, but that is a motivation of the feature which is described in detail in the original proposal https://github.com/metal3-io/metal3-docs/blob/main/design/cluster-api-provider-metal3/node_reuse.md#motivation.